from PyQt5 import QtCore, QtGui, QtWidgets


class Ui_PDF_Merge(object):
    def setupUi(self, Form):
        Form.setObjectName("Form")
        Form.resize(759, 520)
        Form.setMinimumSize(QtCore.QSize(759, 520))
        Form.setMaximumSize(QtCore.QSize(759, 520))
        self.pushButton_select = QtWidgets.QPushButton(Form)
        self.pushButton_select.setGeometry(QtCore.QRect(650, 60, 100, 51))
        font = QtGui.QFont()
        font.setFamily("Gotham Narrow")
        font.setPointSize(14)
        self.pushButton_select.setFont(font)
        self.pushButton_select.setObjectName("pushButton_select")
        self.label_select_file = QtWidgets.QLabel(Form)
        self.label_select_file.setGeometry(QtCore.QRect(360, 80, 151, 16))
        font = QtGui.QFont()
        font.setFamily("Gotham Narrow")
        font.setPointSize(14)
        self.label_select_file.setFont(font)
        self.label_select_file.setObjectName("label_select_file")
        self.line = QtWidgets.QFrame(Form)
        self.line.setGeometry(QtCore.QRect(350, 40, 611, 16))
        self.line.setFrameShape(QtWidgets.QFrame.HLine)
        self.line.setFrameShadow(QtWidgets.QFrame.Sunken)
        self.line.setObjectName("line")
        self.pushButton_merge = QtWidgets.QPushButton(Form)
        self.pushButton_merge.setGeometry(QtCore.QRect(360, 370, 100, 51))
        font = QtGui.QFont()
        font.setFamily("Gotham Narrow")
        font.setPointSize(14)
        self.pushButton_merge.setFont(font)
        self.pushButton_merge.setObjectName("pushButton_merge")
        self.main_label = QtWidgets.QLabel(Form)
        self.main_label.setGeometry(QtCore.QRect(350, 10, 401, 21))
        font = QtGui.QFont()
        font.setFamily("Gotham Narrow")
        font.setPointSize(24)
        self.main_label.setFont(font)
        self.main_label.setAlignment(QtCore.Qt.AlignCenter)
        self.main_label.setObjectName("main_label")
        self.line_2 = QtWidgets.QFrame(Form)
        self.line_2.setGeometry(QtCore.QRect(350, 120, 611, 16))
        self.line_2.setFrameShape(QtWidgets.QFrame.HLine)
        self.line_2.setFrameShadow(QtWidgets.QFrame.Sunken)
        self.line_2.setObjectName("line_2")
        self.label_status = QtWidgets.QLabel(Form)
        self.label_status.setGeometry(QtCore.QRect(360, 440, 391, 16))
        font = QtGui.QFont()
        font.setFamily("Gotham Narrow")
        font.setPointSize(14)
        font.setItalic(False)
        self.label_status.setFont(font)
        self.label_status.setObjectName("label_status")
        self.pushButton_save = QtWidgets.QPushButton(Form)
        self.pushButton_save.setGeometry(QtCore.QRect(650, 370, 100, 51))
        font = QtGui.QFont()
        font.setFamily("Gotham Narrow")
        font.setPointSize(14)
        self.pushButton_save.setFont(font)
        self.pushButton_save.setObjectName("pushButton_save")
        self.placeholder = QtWidgets.QLabel(Form)
        self.placeholder.setGeometry(QtCore.QRect(10, 10, 331, 491))
        font = QtGui.QFont()
        font.setFamily("Gotham Narrow")
        font.setPointSize(12)
        self.placeholder.setFont(font)
        self.placeholder.setFrameShape(QtWidgets.QFrame.Box)
        self.placeholder.setAlignment(QtCore.Qt.AlignCenter)
        self.placeholder.setObjectName("placeholder")
        self.radioButton_onizle = QtWidgets.QRadioButton(Form)
        self.radioButton_onizle.setGeometry(QtCore.QRect(360, 480, 99, 20))
        font = QtGui.QFont()
        font.setFamily("Gotham Narrow")
        font.setPointSize(14)
        self.radioButton_onizle.setFont(font)
        self.radioButton_onizle.setObjectName("radioButton_onizle")
        self.listView = QtWidgets.QListView(Form)
        self.listView.setGeometry(QtCore.QRect(350, 140, 401, 131))
        self.listView.setObjectName("listView")
        self.pushButton_up = QtWidgets.QPushButton(Form)
        self.pushButton_up.setGeometry(QtCore.QRect(360, 280, 100, 51))
        font = QtGui.QFont()
        font.setFamily("Gotham Narrow")
        font.setPointSize(14)
        self.pushButton_up.setFont(font)
        self.pushButton_up.setObjectName("pushButton_up")
        self.pushButton_remove = QtWidgets.QPushButton(Form)
        self.pushButton_remove.setGeometry(QtCore.QRect(650, 280, 100, 51))
        font = QtGui.QFont()
        font.setFamily("Gotham Narrow")
        font.setPointSize(14)
        self.pushButton_remove.setFont(font)
        self.pushButton_remove.setObjectName("pushButton_remove")
        self.pushButton_down = QtWidgets.QPushButton(Form)
        self.pushButton_down.setGeometry(QtCore.QRect(470, 280, 100, 51))
        font = QtGui.QFont()
        font.setFamily("Gotham Narrow")
        font.setPointSize(14)
        self.pushButton_down.setFont(font)
        self.pushButton_down.setObjectName("pushButton_down")

        self.retranslateUi(Form)
        QtCore.QMetaObject.connectSlotsByName(Form)

    def retranslateUi(self, Form):
        _translate = QtCore.QCoreApplication.translate
        Form.setWindowTitle(_translate("Form", "PDF Birleştir"))
        self.pushButton_select.setText(_translate("Form", "PDF Seç"))
        self.label_select_file.setText(_translate("Form", "Lütfen dosyanızı seçiniz:"))
        self.pushButton_merge.setText(_translate("Form", "Birleştir"))
        self.main_label.setText(_translate("Form", "PDF Birleştir"))
        self.label_status.setText(_translate("Form", "Başlamak için bir PDF dosyası seçin..."))
        self.pushButton_save.setText(_translate("Form", "Kaydet"))
        self.placeholder.setText(_translate("Form", "Önizleme için sağ alttaki butonu aktif hale getiriniz."))
        self.radioButton_onizle.setText(_translate("Form", "Önizle"))
        self.pushButton_up.setText(_translate("Form", "Yukarı"))
        self.pushButton_remove.setText(_translate("Form", "Kaldır"))
        self.pushButton_down.setText(_translate("Form", "Aşağı"))
